ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__ota-multi-stable-4.21-upgrade-from-stable-4.21.yaml

```diff
@@ -0,0 +1,87 @@
+base_images:
+  ansible:
+    name: "4.21"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.21"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.21"
+    namespace: ocp
+    tag: openstack-installer
+  tools:
+    name: "4.21"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.21"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+releases:
+  intermediate1:
+    prerelease:
+      architecture: multi
+      product: ocp
+      relative: 1
+      version_bounds:
+        lower: 4.21.0-0
+        stream: 4-dev-preview
```

Comment:
it should be [4-dev-preview-multi](https://multi.ocp.releases.ci.openshift.org/#4-dev-preview-multi), right?

Reply:
@JianLi-RH able to fetch with "4-dev-preview" in multi channel